src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp

```diff
@@ -256,19 +256,16 @@ real64 CompositionalMultiphaseFVM::scalingForSystemSolution( DomainPartition con
 {
   GEOSX_MARK_FUNCTION;
 
+
   // check if we want to rescale the Newton update
-  if( m_maxCompFracChange >= 1.0 )
+  if( m_maxCompFracChange >= 1.0 ||
+      m_maxRelativePresChange >= 1.0 ||
+      ( m_maxRelativeTempChange >= 1.0 || !m_isThermal ) )
```

Comment:
Yes, done. I added some named boolean variables to clarify what I am trying to do:
https://github.com/GEOSX/GEOSX/blob/69e0c7e56e36e755218fc5e7d2adcfc24e504d85/src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp#L259-L268
and thanks for your comment, because there was a bug in my initial check